src/pal/src/loader/module.cpp

```diff
@@ -676,7 +676,7 @@ PAL_RegisterModule(
 {
     HINSTANCE hinstance = nullptr;
 
-    int err = PAL_InitializeDLL();
+    int err = PAL_InitializeDLLWithoutSEH();
```

Comment:
The dac module (mscordacore) does depend on the DllMain handler being called when loaded which does call PAL_InitializeDLL() which is left over from the days when PAL_RegisterModule wasn't being called/existed.

I'll back off on my statement and say now that we can still leave the DllMain handlers in and being called, but remove PAL_RegisterModule as long as none of the DllMain handlers need to use the module handle passed on DLL_PROCESS_ATTACH. mscordbi doesn't need it but it looks like mscordaccore saves the handle away for the ISOSDacInterface::GetDacModuleHandle function.